analyzer/src/main/java/org/jboss/gm/analyzer/alignment/AlignmentTask.java

```diff
@@ -269,9 +280,17 @@ private void writeGmeConfigMarkerFile() throws IOException {
                                 dep.toString(), configuration.getName());
                         return;
                     }
+                    String version = dep.getModuleVersion(); // this is the resolved version from gradle
+                    // if the dependency is present in any of the lockfiles, then we use that version
+                    for (ProjectVersionRef lockFileDep : lockFileDeps) {
+                        if (lockFileDep.asProjectRef().getGroupId().equals(dep.getModuleGroup())
+                                && lockFileDep.asProjectRef().getArtifactId().equals(dep.getModuleName())) {
```

Comment:
I don't think we need to convert to `asProjectRef` just to extract the groupId/artifactId.

Reply:
Good catch

Reply:
Fixed